docs/contributing.md

```diff
@@ -4,4 +4,5 @@ short_title: Contribution Guide
 ---
 
 ```{include} ../CONTRIBUTING.md
+:start-after: "<!-- MYST START -->"
 ```
```

Comment:
````suggestion
% start-after doesn't currently work with {include} so this will just include the whole file
% Tracking issue: https://github.com/executablebooks/mystmd/issues/1096
```{include} ../CONTRIBUTING.md
:start-after: "<!-- MYST START -->"
```
````

Reply:
@rowanc1 what do you think about adding this comment for context and then just merging this one?

Reply:
Yep, that works!